packages/synapse-react-client/src/components/UserCard/UserCardMedium.tsx

```diff
@@ -198,7 +190,7 @@ export function UserCardMedium({
             tabIndex={0}
           >
             <span style={{ paddingRight: '5px', paddingBottom: '2px' }}>
-              <a className={`link ${isLarge ? 'SRC-whiteText' : ''}`}>
+              <a className={`link ${isLarge ? 'SRC-blackText' : ''}`}>
```

Comment:
Remove these from other places -- unless it's still necessary
```suggestion
              <a className={`link`}>
```